config/initializers/recaptcha.rb

```diff
@@ -0,0 +1,6 @@
+Recaptcha.configure do |config|
+  config.site_key  = "6LfsUJsUAAAAAMM7jfji6WdJjg1QHUr8IAqdt-Uj"
+  config.secret_key = "6LfsUJsUAAAAABvi62pFJu8-cPVT1VludSwMgW4X"
+  # Uncomment the following line if you are using a proxy server:
+  # config.proxy = 'http://myproxy.com.au:8080'
+end
```

Comment:
It can be quite dangerous to post secret keys onto GitHub, so I can't merge this PR as-is. 

Adding these as environment variables is probably the most correct solution, but we can do this another way that should be easier. 

1. Add this file to your `.gitignore` file. 
2. Create a new file named `config/initializers/recaptcha_example.rb`.
3. In this new file, copy and paste the content from this file, but replace both keys with something like `"example"`. 
4. You will have to actually send your teammates the ignored file in order for the code to work for them.